services/certificate-api/src/routes/certificate_controller.js

```diff
@@ -103,20 +104,26 @@ async function generateRawCertificate(certificate, templateUrl, entityId) {
     let certificateRaw = certificate;
     // TODO: based on type template will be picked
     const certificateTemplateUrl = templateUrl;
-    const zip = new JSZip();
-    zip.file("certificate.json", certificateRaw, {
-        compression: "DEFLATE"
-    });
-    const zipType = (envData.qrType && envData.qrType.toUpperCase() === URL_BASED_QR);
-    const zippedData = await zip.generateAsync({type: zipType ? 'base64': 'string', compression: "DEFLATE"})
-        .then(function (content) {
-            // console.log(content)
-            return content;
+    const qrCodeType = envData.qrType || '';
+    let qrData;
+    console.log('QR Code type: ', qrCodeType);
+    if (qrCodeType.toUpperCase() === URL) {
+        qrData = `${envData.certDomainUrl}/certs/${entityId}?t=${qrCodeType}`;
```

Comment:
you can make certs part of the config so that we dont enforce /cert is required.